ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ota-multi-stable-4.16-upgrade-from-stable-4.15.yaml

```
@@ -0,0 +1,85 @@
+base_images:
+  ansible:
+    name: "4.15"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.15"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.16"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.15"
+  tools:
+    name: "4.15"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.15"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    release:
+      architecture: multi
+      channel: fast
+      version: "4.15"
+  target:
+    prerelease:
+      architecture: multi
+      product: ocp
+      version_bounds:
+        lower: 4.16.0-0
+        upper: 4.17.0-0
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: azure-ipi-amd-mixarch-f28
+  cron: 26 14 16 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      ADDITIONAL_WORKER_VM_TYPE: Standard_D4ps_v5
+      ADDITIONAL_WORKERS: "1"
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "2"
+      ENABLE_OTA_TEST: OCP-53907
+    test:
+    - ref: ipi-install-heterogeneous
+    - ref: cucushift-upgrade-healthcheck
+    - ref: cucushift-upgrade-setchannel
+    - ref: cucushift-ota-preupgrade
+    - ref: cucushift-upgrade-toversion
+    - chain: cucushift-installer-check-cluster-health
```

cucushift-upgrade-healthcheck```suggestion
    - ref: cucushift-upgrade-healthcheck
```
Last time, I pointed to a wrong health check step name, maybe `cucushift-upgrade-healthcheck` is more proper here. Sorry for that.

---

there is already `cucushift-upgrade-healthcheck` in line74, dup steps name seems not accepted here.

---

Ah, I did not noticed the line 74, we also have `cucushift-upgrade-prehealthcheck` to use, I guess we can use it on line 74. For now, I saw you switch the place of `cucushift-upgrade-healthcheck` and `cucushift-installer-check-cluster-health`, that is also a solution for me.